src/configure/resources/messages.ts

```diff
@@ -52,4 +52,11 @@ export class Messages {
     public static azureServicePrincipalFailedMessage: string =`Failed while creating Azure service principal.`;
     public static roleAssignmentFailedMessage: string =`Failed while role assignement.`;
     public static waitForAzureSignIn: string =`Waiting for Azure sign-in...`;
+    public static userCancelledExcecption = 'User cancelled the action';
+    public static deploymentCenterAlreadyConfigured = 'Deployment source of type: %s is already setup for your web app, you can view more by browsing to deployment center.';
```

Comment:
A pipeline is already setup for this Azure Web App.  Please click on browse to explore